claim/claim_test.go

```diff
@@ -195,33 +193,16 @@ func TestMarshal_AllFields(t *testing.T) {
 }
 
 func TestClaimSchema(t *testing.T) {
-	t.Skip("This test is pending an alternate, offline-friendly implementation; see https://github.com/cnabio/cnab-go/issues/194")
+	t.Skip("This test is pending non-trivial updates to the Claim and Result objects: https://github.com/cnabio/cnab-go/issues/202")
```

Comment:
Note this test is currently still being skipped.  Quite a few changes to the Claim and Result objects are coming down the pike and will need a follow-up to update and start running this test.